src/PresentationalComponents/RulesTable/RulesTable.original.js

```
@@ -0,0 +1,451 @@
+          perPage: Number(filters.limit),
+          onSetPage(_event, page) {
+            onSetPage(page);
+          },
+          onPerPageSelect(_event, perPage) {
+            setFilters({ ...filters, limit: perPage, offset: 0 });
+          },
+          isCompact: true,
+        }}
+        exportConfig={
+          envContext.isExportEnabled && {
+            label: intl.formatMessage(messages.exportCsv),
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)